src/js/features/image-generation/media-modal/views/prompt.js

```diff
@@ -35,17 +35,31 @@ const Prompt = wp.media.View.extend( {
 	 */
 	promptRequest: function ( event ) {
 		let prompt = '';
+		let quality = '';
+		let size = '';
+		let style = '';
 
 		if ( event.which === 13 ) {
 			prompt = event.target.value.trim();
 		} else if ( event.target.nodeName === 'BUTTON' ) {
 			prompt = event.target.parentElement
 				.querySelector( '.prompt' )
 				.value.trim();
+			quality = event.target.parentElement
+				.querySelector( '.quality' )
+				.value.trim();
+
+			size = event.target.parentElement
+				.querySelector( '.size' )
+				.value.trim();
+
+			style = event.target.parentElement
+				.querySelector( '.style' )
+				.value.trim();
```

Comment:
Because we are now using `event.target.parentElement` multiple times, we should store that in a variable instead of getting that element over and over